src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+function Button({ name }) {
+  return (
+    <div>
+      <button type="button">{name}</button>
+    </div>
+  );
+}
+
+Button.propTypes = {
+    name: propTypes.string, // eslint-disable-line
+};
```

Comment:
- [ ] Eslint is complaining here because the `name` prop was declared as a not required prop. Whenever you declare a prop as not required, we must provide a default value for it (the same way you did in the Display component for the `result` prop). We can safely fix this offense by informing that this prop is required.
```JS
Button.propTypes = {
    name: propTypes.string.isRequired,
}
```